src/AutoRest.CSharp.V3/Input/CodeModelPartials.cs

```diff
@@ -95,6 +95,11 @@ public ObjectSchema()
         public bool IsOutput => Usage.Contains(SchemaContext.Output);
     }
 
+    // redefined manually to inherit from ObjectSchema
+    internal partial class GroupSchema : ObjectSchema
```

Comment:
Would it make sense to make an issue for M4 to update it to do this?

Reply:
Not sure, it's not like what they are doing is wrong, it's just more convenient for us to parse it this way.